voila/handler.py

```diff
@@ -21,6 +23,8 @@
 
 class VoilaHandler(JupyterHandler):
 
+    executor = ThreadPoolExecutor()
```

Comment:
From 3.5+, if not specified, `max_workers` defaults to "to the number of processors on the machine, multiplied by 5".

https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.ThreadPoolExecutor